compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -4286,6 +4311,32 @@ void t_go_generator::parse_go_tags(map<string,string>* tags, const string in) {
   }
 }
 
+void t_go_generator::generate_deprecation_comment(ostream& out, map<string, vector<string>>& annotations) {
```

Comment:
Is there a reason why `annotations` are not const? is there an expectation that it will be modified inside this function at some time?
```suggestion
void t_go_generator::generate_deprecation_comment(ostream& out, const map<string, vector<string>>& annotations) {
```